webapp/channels/src/components/admin_console/admin_definition.jsx

```diff
@@ -5727,6 +5727,16 @@ const AdminDefinition = {
                         key: 'GuestAccountsSettings.Enable',
                         isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.AUTHENTICATION.GUEST_ACCESS)),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'GuestAccountsSettings.HideTags',
+                        label: t('admin.guest_access.hideTags'),
+                        label_default: 'Hide guest tag',
+                        help_text: t('admin.guest_access.hideTagsDescription'),
+                        help_text_default: 'When true, the \'guest\' tag will not be shown next to the name of all guest users in the Mattermost chat interface.',
```

Comment:
Minor nit, but I think this should be `"guest"` instead of `'guest'`.

Also, you forgot to add this to `en.json` which is what CI is complaining about